src/bin/readstat.c

```diff
@@ -381,7 +402,11 @@ static int convert_file(const char *input_filename, const char *catalog_filename
             fprintf(stderr, "Error opening %s: File exists (Use -f to overwrite)\n", output_filename);
         } else {
             fprintf(stderr, "Error processing %s: %s\n", rs_ctx->error_filename, readstat_error_message(error));
+#if !defined _MSC_VER
             unlink(output_filename);
+#else
+            _unlink(output_filename);
+#endif
```

Comment:
For platform differences like this, I would prefer to move the logic somewhere else so that it can be reused. For a simple case like this probably adding to the top of the file

```C
#if defined _MSC_VER
#define unlink _unlink
#endif
```

Would be fine.